LibSWBF2/Chunks/BaseChunk.cpp

```diff
@@ -162,7 +162,7 @@ namespace LibSWBF2::Chunks
 		else
 		{
 			// if we've got trailing bytes, skip them too
-			//ForwardToNextHeader(stream);
+			ForwardToNextHeader(stream);
```

Comment:
Feel free to comment this back out, I'm not sure why I added back in, but I'm also not sure why it was commented out.  Isn't it harmless/inert?

Reply:
I avoid to use it, since it implies there's something inconsistent in our chunk reading process. If we end up somewhere arbitrary after reading the last chunk and need to crawl our way through unknown data (which will be left uninterpreted) until we find the next header, something is off.

Reply:
Gotcha